src/RulesetEngine/SkippedViolationHelper.php

```diff
@@ -25,15 +27,12 @@ public function __construct(ConfigurationSkippedViolation $configuration)
         $this->unmatchedSkippedViolation = $configuration->all();
     }
 
-    public function isViolationSkipped(ClassLikeName $classLikeNameA, ClassLikeName $classLikeNameB): bool
+    public function isViolationSkipped(string $dependant, string $dependee): bool
```

Comment:
I prefer the original implementation. Now instead of calling `toString()` once in here each callee of this method will do the `toString()` themselves.